### PR TITLE
Issue #139: modified log method for JIG and Mongo.

### PR DESCRIPTION
**Pull request by Mindiell**
Just a light modification as for sql.

### CODE REVIEW

**Comment by ikkez** [comment]
`$flag` is not defined

**Comment by Mindiell** [comment]
What a dumb PR I made :D
Thx for the review…

File: db/jig.php [hunk 1]
```diff
@@ -109,7 +109,7 @@ function uuid() {
 	*	Return profiler results
 	*	@return string
 	**/
-	function log() {
+	function log($flag=TRUE) {
```
Comment (Rayne) [inline db/jig.php]: Please update the documentation.

File: db/mongo.php [hunk 2]
```diff
@@ -60,7 +60,7 @@ function uuid() {
 	*	Return MongoDB profiler results
 	*	@return string
 	**/
-	function log() {
+	function log($flag=TRUE) {
```
Comment (Rayne) [inline db/mongo.php]: Please update the documentation.

**Comment by Mindiell** [comment]
Done.

**Pull request merged**